javascript/javascript1/week3/travel_time.js

```diff
@@ -0,0 +1,14 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function timeInfo(){
+    const time = travelInformation.destinationDistance / travelInformation.speed;
+    const hours = Math.floor(time);
+    const min = Math.floor((time - hours) * 100);
```

Comment:
why not 60 ? since 1h is 60 minutes.
```suggestion
    const min = Math.floor((time - hours) * 60);
```

Reply:
yes implemented the change